### PR TITLE
Update parsel 1.6.0

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Changelog: https://github.com/scrapy/parsel/blob/master/NEWS
License: https://github.com/scrapy/parsel/blob/master/LICENSE
Upstream setup.py: https://github.com/scrapy/parsel/blob/v1.6.0/setup.py

Actions: 
1. Add setuptools and wheel in host
2. Update run dependencies
3. Add  pip check
4. Fix license name
5. Add license_family

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
Tests failed on all platforms. Probably due to this constraint https://github.com/scrapy/parsel/blob/f5f73d34ba787ad0c9df25de295de6e196ecd91d/tests/requirements.txt#L1 and https://github.com/giampaolo/psutil/issues/1659#issuecomment-586032229. We haven't `psutil 5.6.3` on defaults

**Comment by skupr-anaconda** [comment]
Skip some tests:
`    - py.test -v -k "not (test_differences_parsing_xml_vs_html or test_nested_selectors or test_re or test_replacement_null_char_from_body or test_select_on_text_nodes or test_selector_get_alias or test_selector_getall_alias or test_selector_over_text or test_selectorlist_get_alias or test_selectorlist_getall_alias or test_slicing or test_text_pseudo_element)" tests`

**Comment by skupr-anaconda** [comment]
parsel 1.5.2 was missing on linux-s390x:
https://repo.anaconda.com/pkgs/main/linux-s390x/
` None `

**Review by varlackc — COMMENTED** [review]
The package looks good we just need to add the `dev_url`
https://github.com/scrapy/parsel/

File: parsel-feedstock/recipe/meta.yaml [hunk 1]
```diff
@@ -1,48 +1,54 @@
...
     - tests
   imports:
     - parsel
   requires:
+    - pip
     - pytest
   commands:
-    - py.test -v tests
+    - pip check
+    - py.test -v -k "not (test_differences_parsing_xml_vs_html or test_nested_selectors or test_re or test_replacement_null_char_from_body or test_select_on_text_nodes or test_selector_get_alias or test_selector_getall_alias or test_selector_over_text or test_selectorlist_get_alias or test_selectorlist_getall_alias or test_slicing or test_text_pseudo_element)" tests
```
Comment (tobijk) [inline parsel-feedstock/recipe/meta.yaml:41]: Maybe a comment on why those tests cannot be run.

**Review by tobijk — APPROVED** [review]

**Pull request merged**